### PR TITLE
Feature/nunit3 add possibility to use fullname attribute as test name

### PR DESCRIPTION
**Pull request by VladislavKostyukevich**
# PR Details

Added possibility To choose test name node before importing NunitV3 file (Feature: Test Name will be used by default)

## Related Issue

#27 

## How Has This Been Tested

1. Import Nunit v3 file with Feature: Test Name option
1. See the imported test names
1. Import Nunit v3 file with Test Class Name option
1. See the imported test names

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: e2e/specs/import/importNunit3.spec.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        await projectList.removeProject(project.name);
+        if (await projectList.menuBar.isLogged()) {
+            return projectList.menuBar.clickLogOut();
+        }
+    });
+
+    it('NUnit v3 option can be selected and Files can be uploaded', async () => {
+        await projectView.menuBar.import();
+        await importPage.selectImportType(importPage.importTypes.nunitV3);
+        await importPage.uploadFile(testData.getFullPath(importFiles.nunitV3));
```
Comment (knysh) [inline e2e/specs/import/importNunit3.spec.ts]: As I understand, you missed 'expect'.

File: e2e/specs/import/importNunit3.spec.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        await testRunList.filterByBuildName(featureName.buildName);
+        return expect(testRunList.getTestRunsCount()).toBe(1, 'Should be only one test run in table!');
+    });
+
+    it('Results were correctly created during Feature: TestName import results via UI', async () => {
+        await testRunList.openTestRun(featureName.buildName);
+        await testRunView.sortResultsByName();
+        const actualResults = await testRunView.getResultsCSV();
+        const expected = await testData.readAsString('/resultsTable/nunitV3FeatureName.csv');
+        return expect(actualResults).toBe(expected);
```
Comment (knysh) [inline e2e/specs/import/importNunit3.spec.ts]: add friendly message please

File: e2e/specs/import/importNunit3.spec.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        await testRunList.filterByBuildName(className.buildName);
+        return expect(testRunList.getTestRunsCount()).toBe(1, 'Should be only one test run in table!');
+    });
+
+    it('Results were correctly created during Class Name import results via UI', async () => {
+        await testRunList.openTestRun(className.buildName);
+        await testRunView.sortResultsByName();
+        const actualResults = await testRunView.getResultsCSV();
+        const expected = await testData.readAsString('/resultsTable/nunitV3ClassName.csv');
+        return expect(actualResults).toBe(expected);
```
Comment (knysh) [inline e2e/specs/import/importNunit3.spec.ts]: add friendly message please

File: e2e/utils/testData.util.ts [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+                        fs.unlinkSync(curPath);
+                    }
+                });
+                resolve();
+            }
+        });
+    }
+
+    /**
+     * Check if file exists
```
Comment (knysh) [inline e2e/utils/testData.util.ts]: maybe "Wait for file exists"?

File: e2e/pages/import.po/helpers.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+class ImportHelper {
+    getTestNameSwitcher(testNameType: string): UiSwitch {
+        switch (testNameType) {
+            case testNameTypes.className:
+                return elements.classNameSwitch;
+            case testNameTypes.featureTestName:
+                return elements.featureTestNameSwitch;
+            case testNameTypes.testDescription:
+                return elements.testDescriptionSwitch;
+            case testNameTypes.testName:
```
[the remark this answers is not in the document]
Comment (VladislavKostyukevich) [reply e2e/pages/import.po/helpers.ts]: Fixed

File: e2e/pages/testrun/view.po/index.ts [hunk 6]
```diff
@@ -88,9 +88,16 @@ export class TestRunView extends BasePage {
...
+  async getResultsCSV() {
+    return elements.resultsTable.getCSV();
+  }
+
   async resultsAreFiltered(column: string, value: string): Promise<boolean> {
     const isSelected = await elements.resultsTable.isFilterSelected(column, value);
     const isFiltered = await elements.resultsTable.isContainOnlyRowsWith(column, value);
     return isSelected && isFiltered;
   }
+  sortResultsByName() {
```
[the remark this answers is not in the document]
Comment (VladislavKostyukevich) [reply e2e/pages/testrun/view.po/index.ts]: Fixed

File: e2e/specs/import/importNunit3.spec.ts (hunk 2 above)
Comment (VladislavKostyukevich) [reply e2e/specs/import/importNunit3.spec.ts]: Fixed

File: e2e/specs/import/importNunit3.spec.ts (hunk 3 above)
Comment (VladislavKostyukevich) [reply e2e/specs/import/importNunit3.spec.ts]: Fixed

File: e2e/utils/testData.util.ts (hunk 4 above)
Comment (VladislavKostyukevich) [reply e2e/utils/testData.util.ts]: Fixed

File: e2e/specs/import/importNunit3.spec.ts (hunk 1 above)
Comment (VladislavKostyukevich) [reply e2e/specs/import/importNunit3.spec.ts]: Fixed

**Pull request merged**